docs/students/faqs.md

```diff
@@ -7,9 +7,9 @@ title: Frequently Asked Questions
 
 ### Is this course recognised by employers?
 
-Most jobs don't ask for a degree, however, more and more companies recognize CYF courses as a Full Stack Development bootcamp as bootcamps in general are becoming a more popular way to study as a developer.
+Most jobs don't ask for a degree, however, more and more companies recognize the CYF course as a Full Stack Development bootcamp. Bootcamps in general are becoming a more popular way to study as a developer.
 
-### Do we have any Saturday’s off during school half term break or Easter or during the summer?
+### Do we have any Saturdays off during school half term break, Easter or during the summer?
```

Comment:
Change to "any days off" to account for all schools. (we teach on Sundays in London)

Reply:
Good spot! I also removed other references to Saturdays